iron-list.html

```diff
@@ -938,10 +961,9 @@
      */
     _increasePoolIfNeeded: function() {
       var self = this;
-      var isClientFull = this._physicalBottom >= this._scrollBottom &&
-          this._physicalTop <= this._scrollPosition;
-
-      // Base case 2: if the physical size is optimal and the list's client height is full
+      var isClientFull = this._physicalBottom + this._scrollOffset >= this._scrollBottom &&
+          this._physicalTop <= Math.max(0, this._scrollPosition - this._scrollOffset);
```

Comment:
Not sure if I completely understand this part - why is the condition for `_physicalTop` bounded by 0 but not `_physicalBottom`?